tests/_util.py

```diff
@@ -67,13 +65,3 @@ def disable_asgi_non_coroutine_wrapping():
 
     if should_wrap:
         os.environ['FALCON_ASGI_WRAP_NON_COROUTINES'] = 'Y'
-
-
-def as_params(*values, prefix=None):
```

Comment:
This wasn't as easy to reuse as fixture, but I reduced it to a oneliner, and it is used only in two places.
So I went on to simply inline it where affected.